content/typescript/typescript.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: 'TypeScript'
+Codecademy Hub Page: null
+CatalogContent:
+  - 'learn-typescript'
+  - 'paths/web-development'
+---
+
+TypeScript is a typed superset of JavaScript that combines the flexibility and general syntax of JavaScript with a type checker and syntax.
+It's one of the fastest-growing and most popular languages on the web today.
+
+Code written in TypeScript compiles to regular JavaScript.
+Anything you write in TypeScript can be run wherever it could be in JavaScript: web browsers, Node servers, and more.
+
+## What is TypeScript?
```

Comment:
We usually just want one or two paragraphs here. But let's merge this, because you just gave me an idea, to add an About.md for each language. I can do this after merging.

Reply:
Nifty. We can also make the backend automatically sort `About` pages first!